Gemfile

```diff
@@ -4,23 +4,14 @@ ruby '2.3.3'
 
 gem 'actionmailer-text'
 gem 'active_model_serializers'
-gem 'bourbon', '5.0.0.beta.5'
-gem 'coffee-rails', '~> 4.1.0'
```

Comment:
🔥 🔥 🔥 🔥 

I believe at one point our zxcvbn code relied on that? But if the tests pass.... 💯